Microsoft.Azure.CosmosRepository/src/DefaultRepository.cs

```diff
@@ -279,6 +279,46 @@ public async ValueTask<bool> ExistsAsync(Expression<Func<TItem, bool>> predicate
             return count > 0;
         }
 
+        public async ValueTask<IPage<TItem>> ScrollAsync(
+            int lastPage = 0,

```

Comment:
This parameter seems a bit odd - it seems to only be incremented and integrated in the returned Page (no actual effect on the query etc). I'm not sure I'd do this rather than leaving it up to the API user to track which page their own (at least with the current API shape) - but see my general comment about an API which streams pages.

Reply:
Yes, this did seem a little odd I suppose we could remove this parameter and like you say let the caller keep track of the current page. As this does not really serve any purpose here.

Reply:
Removed this parameters as it did not provide much benefit 👍